docs/telemetry.md

```diff
@@ -4,10 +4,24 @@ Fenix uses Mozilla's telemetry service (Glean) and LeanPlum to measure feature p
 
 ## Baseline ping
 
-Fenix creates and tries to send a "baseline" ping when the app goes to the background. This baseline ping is defined by the [Glean](https://github.com/mozilla-mobile/android-components/tree/master/components/service/glean) component and [documented in the Android Components repository](https://github.com/mozilla-mobile/android-components/blob/master/components/service/glean/docs/baseline.md).
+Fenix creates and tries to send a "baseline" ping when the app goes to the background. This baseline ping is defined by the [Glean](https://github.com/mozilla-mobile/android-components/tree/master/components/service/glean) component and [documented in the Android Components repository](https://github.com/mozilla-mobile/android-components/blob/master/components/service/glean/docs/pings/baseline.md).
+
+## Metrics ping
+
+Fenix creates and tries to send a "baseline" ping. It is defined inside the [`metrics.yaml`](https://github.com/mozilla-mobile/fenix/blob/master/app/metrics.yaml) file. This ping includes things like wether or not Fenix is currently the default browser.
 
 ## Events
 
-| Event     | Glean Key | Leanplum Key | extras |
-|-----------|-----------|--------------|--------|
-| OpenedApp |           | E_Opened_App |        |
\ No newline at end of file
+Fenix sends event pings that allows us to measure feature performance.
+
+| Event           | Glean Key         | Leanplum Key | extras                |
+|-----------------|-------------------|--------------|-----------------------|
+| OpenedApp       | app_opened        | E_Opened_App | source*               |
+| SearchBarTapped | search_bar_tapped |              | source**              |
+| EnteredUrl      | entered_url       |              | autocomplete***       |
+| PerformedSearch | performed_search  |              | search_suggestion**** |
+
+* `source`: The method used to open Fenix (For exmaple: `app_icon` or `link`)
+** `source`: The view the user was on when they initiated the search (For example: `Home` or `Browser`)
+*** `autocomplete`: A boolean that tells us wether the URL was autofilled by an Autocomplete suggestion
```

Comment:
Eventually we should be able to stop needing to document this twice - the `metrics.yaml` should be enough, and that's the point of Glean, that the yaml is self-documenting.

@mdboom this isn't totally necessary, but is there a way for the `metrics.yaml` to be parsed and turned into an easily human-readable docs page? Ideally something like this wiki page :)

Reply:
Yes, the [probe-scraper](https://github.com/mozilla/probe-scraper) is just such a tool.  I believe it's already running over the reference-browser.  We should reach out to @fbertsch after this is merge to add fenix.